MetaMask/metamask-mobile
# [FIX] SRP recover when an error is thrown

**Description**
The SRP was not possible to recover when an error was found.

**Proposed solution**
SRP is able to be recovered when an error is found

**How to test it**
* Setup dev environment
* Write this code line `throw new Error('testing')` on the beginning of [wallet/index.tsx](https://github.com/MetaMask/metamask-mobile/blob/e05ef5ee785f171ba994373ebde4128e2dd18d20/app/components/Views/Wallet/index.tsx#L72) 
* Reload the app
* Insert password
* Expected to see the Error boundary screen
* Expected to be able to recover password
* (ONLY ON ANDROID)
* Try to take a screenshot of SRP
* Expected behaviour is that it's not possible

(This screen doesn't have the warning alert of when the screenshot is taken on IOS)

**Screenshots/Recordings**
https://recordit.co/dbQHB41jPb

**Issue**

Progresses #

**Checklist**
